src/pkg/packager/common.go

```diff
@@ -456,6 +457,37 @@ func (p *Packager) validatePackageArchitecture() error {
 	return nil
 }
 
+// validateLastNonBreakingVersion compares the Zarf CLI version against a package's LastNonBreakingVersion.
+// It will return an error if there is an error parsing either of the two versions,
+// and will throw a warning if the CLI version is less than the LastNonBreakingVersion.
+func (p *Packager) validateLastNonBreakingVersion() (err error) {
+	cliVersion := config.CLIVersion
+	lastNonBreakingVersion := p.cfg.Pkg.Build.LastNonBreakingVersion
+
+	if lastNonBreakingVersion != "" && cliVersion != "UnknownVersion" {
```

Comment:
nit;

This comes down to just style choices but I think it's easier to read code when it has short-circuiting blocks instead of nested blocks.

Something like this:
```Go
if lastNonBreakingVersion == "" || cliVersion == "UnknownVersion" {
    return nil
}

// TODO: Fill in the main logic of the function.

}

Reply:
Good call out. I agree, using short-circuiting improves readability. I pushed the change